README.md

```diff
@@ -1 +1,85 @@
-###### official website for the #TechMasters Slack Group [techmasters.chat](http://techmasters.chat)
+# Technology Masters Website
+
+The official website for the #TechMasters Slack Group [techmasters.chat](http://techmasters.chat)
+
+The site is published using [Jekyll](https://jekyllrb.com), a Ruby Gem that creates static
+pages that can be hosted anywhere.
+
+## Installation
```

Comment:
There's already a "Manual Installation" heading which can be expanded upon by people who understand Ruby and RVM better than I do.